cmd/podman/containers/cp.go

```diff
@@ -55,10 +55,13 @@ var (
 
 func cpFlags(cmd *cobra.Command) {
 	flags := cmd.Flags()
-	flags.BoolVar(&cpOpts.Extract, "extract", false, "Deprecated...")
-	flags.BoolVar(&cpOpts.Pause, "pause", true, "Deprecated")
+	flags.BoolVar(&cpOpts.OverwriteDirNonDir, "overwrite", false, "Allow to overwrite directories with non-directories and vice versa")
 	flags.BoolVarP(&chown, "archive", "a", true, `Chown copied files to the primary uid/gid of the destination container.`)
+
+	// Deprecated flags: exist for backwards compat
```

Comment:
Should we add a TODO (5.0) remove the options comment?

Reply:
I am no fan of breaking deployments unless we have to.

Reply:
Well arguably the users of this option already had the breaking change when they were changed into NOPs.
Hiding this fact by still accepting this option makes it worse not better IMO.

Reply:
Thanks for pointing that out. I assumed those flags were still plugged.  Since they are NOPs, we can remove them with 5.0.